.github/workflows/downstream-project-spl-nightly.yml

```diff
@@ -0,0 +1,36 @@
+name: Downstream Project - SPL (Nightly)
+
+on:
+  schedule:
+    - cron: "0 3 * * *"
```

Comment:
Can you add me to whatever list gets a notification if the nightly run fails?

Reply:
do you mind if the channel receives not only the nightly run reporting? feel free to let me know if you prefer to have an individual one!